packit_service/worker/jobs.py

```diff
@@ -295,9 +301,37 @@ def __init__(
         self.job: JobConfig = job
         self.triggered_by: JobTriggerType = triggered_by
 
+        self.api: Optional[PackitAPI] = None
+        self.local_project: Optional[PackitAPI] = None
+
+        if not config.actions_handler_work_dir:
+            raise RuntimeError(
+                "Packit service has to run with actions_handler_work_dir set."
+            )
+
+        self._clean_workplace()
+
     def run(self):
         raise NotImplementedError("This should have been implemented.")
 
+    def _clean_workplace(self):
+        logger.debug("remove contents of the PV")
+        p = Path(self.config.actions_handler_work_dir)
+        # remove everything in the volume, but not the volume dir
+        globz = list(p.glob("*"))
+        if globz:
+            logger.info("volume was not empty")
```

Comment:
Please include in the info also self.config.actions_handler_work_dir, so it is clear from log what are you talking about.

Reply:
I'd say it's not super-useful to show the path since the path is set in a config file created by us, so we know it up front what it is.

But I'd say it's a good thing to log that path at least once.

Reply:
Okay, it doesn't make sense to log the handler_work_dir path explicitly since it will be shown in the log below, you mentioned:
```python
In [1]: p = pathlib.Path("/").glob("*")    

In [2]: list(p)                            
Out[2]: 
[PosixPath('/boot'),
 PosixPath('/dev'),
 PosixPath('/proc'),
 PosixPath('/run'),
 PosixPath('/sys'),
 PosixPath('/root'),
...
```